src/main/java/seedu/apollo/ui/Parser.java

```diff
@@ -194,11 +214,44 @@ private static Command parseCommand(String[] split, int size, ModuleList moduleD
             String moduleCode = split[1];
             return new DeleteModuleCommand(moduleCode);
 
+
+
         default:
             throw new IllegalCommandException();
         }
     }
 
+    /**
+     * Method that selects which help command class to invoke
+     * @param helpCommandName
+     * @return The appropriate HelpCommandClass child
+     * @throws IllegalArgumentException If an unknown command is input by the user.
+     */
+    public static HelpCommand chooseHelpCommand(String helpCommandName) throws IllegalArgumentException{
+        switch(helpCommandName) {
+        case "list":
+            return new ListHelpCommand();
+        case "todo":
+            return new TodoHelpCommand();
+        case "deadline":
+            return new DeadlineHelpCommand();
+        case "event":
+            return new EventHelpCommand();
+        case "mark":
+            return new MarkHelpCommand();
+        case "unmark":
+            return new UnmarkHelpCommand();
+        case "delete":
+            return new DeleteHelpCommand();
+        case "find":
+            return new FindHelpCommand();
+        case "date":
+            return new DateHelpCommand();
+        default:
+            throw new IllegalArgumentException("Invalid command name: " + helpCommandName);
```

Comment:
oh ya u already did it here HAHAHAH just move it on top since the if statement is super long